From eb6bf3e38ace9cec60c5ad730f08734104fc89ec Mon Sep 17 00:00:00 2001
From: Junki Kato <junki.kato.xk@renesas.com>
Date: Tue, 11 Sep 2018 19:43:08 +0900
Subject: [PATCH 265/909] usb: gadget: udc: renesas_usb3: add binding for
 r8a77990

commit 3c34ed3034b9c72374493076fcc0b2a847715204 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds binding for r8a77990 (R-Car E3)

Signed-off-by: Junki Kato <junki.kato.xk@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/devicetree/bindings/usb/renesas_usb3.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/usb/renesas_usb3.txt b/Documentation/devicetree/bindings/usb/renesas_usb3.txt
index 2c071bb5801e..4569da81cedd 100644
--- a/Documentation/devicetree/bindings/usb/renesas_usb3.txt
+++ b/Documentation/devicetree/bindings/usb/renesas_usb3.txt
@@ -5,6 +5,7 @@ Required properties:
 	- "renesas,r8a7795-usb3-peri"
 	- "renesas,r8a7796-usb3-peri"
 	- "renesas,r8a77965-usb3-peri"
+	- "renesas,r8a77990-usb3-peri"
 	- "renesas,rcar-gen3-usb3-peri" for a generic R-Car Gen3 compatible
 	  device
 
-- 
2.17.1

